captcha/templates/captcha/includes/js_v2_invisible.html

```diff
@@ -1,19 +1,21 @@
 {# The provided implementation caters for only one reCAPTCHA on a page. Override this template and its logic as needed. #}
-<script src="https://www.google.com/recaptcha/api.js" async defer></script>
+<script src="https://www.google.com/recaptcha/api.js{% if language %}?hl={{ language }}{% endif %}"></script>
```

Comment:
Should we add `async` to the `script` definition here?

Reply:
As per external discussion, JavaScript and HTML improvements can be done at a later date.